pkg/customers/approval.go

```diff
@@ -129,34 +136,43 @@ func updateCustomerAddress(logger log.Logger, repo CustomerRepository) http.Hand
 	return func(w http.ResponseWriter, r *http.Request) {
 		w = route.Responder(logger, w, r)
 
-		if r.Method != "PUT" {
-			moovhttp.Problem(w, fmt.Errorf("unsupported HTTP verb %s", r.Method))
-			return
-		}
-
 		customerID, addressId := route.GetCustomerID(w, r), getAddressId(w, r)
 		if customerID == "" || addressId == "" {
 			return
 		}
+		requestID := moovhttp.GetRequestID(r)
 
-		var req updateCustomerAddressRequest
-		if err := json.NewDecoder(r.Body).Decode(&req); err != nil {
-			moovhttp.Problem(w, err)
-			return
-		}
-		if err := req.validate(); err != nil {
-			moovhttp.Problem(w, err)
-			return
-		}
+		switch r.Method {
```

Comment:
The handlers are setup differently in `admin` so I have to add this `switch` to include a `DELETE` on the url. I'm hoping we can refactor the handler code in the future.

Reply:
Fixed.